DIFFERENCES.md

```diff
@@ -26,3 +26,16 @@
   This is a bug and it's already fixed in the Go version.
 - With --debug the Go version sends all debug output to stderr. The Ruby
   version sends http debug to stderr, and other debug to stdout.
+- In zypper-migration plugin, bool flag arguments matching default value are
+  ignored. This changes behavior for cases where both positive and negative
+  flag is passed, e.g.: `zypper migration --verbose --no-verbose` disables
```

Comment:
so this should not be possible anymore with the contradicts checker

Reply:
removed